api_docs/tf_quant_finance/rates.md

```diff
@@ -37,7 +37,7 @@ Functions to handle rates.
 
 ## Classes
 
-[`class SwapCurveBuilderResult`](../tf_quant_finance/rates/SwapCurveBuilderResult.md): SwapCurveBuilderResult(times, rates, discount_factors, initial_rates, converged, failed, iterations, objective_value)
+[`class SwapCurveBuilderResult`](../tf_quant_finance/rates/SwapCurveBuilderResult.md): Build curves for swapping future cash flows.
```

Comment:
The docs are auto-generated from the inline code documentation. So the right way to fix this would be to modify the docstring of class SwapCurveBuilderResult defined in swap_curve_common.py.

SwapCurveBuilderResult is a namedtuple so we need to add the docstring in a roundabout way. My suggestion would be to do:

SwapCurveBuilderResult.__doc__ = """Results from a swap curve construction ..."""